pyproject.toml

```diff
@@ -34,6 +34,7 @@ more-itertools = "^9.1.0"
 django-cors-headers = "^4.3.1"
 
 [tool.poetry.group.dev.dependencies]
+pip = "23.3.1"  # TODO BWS Add explanation
```

Comment:
Yes, I know. I have a local commit that I have not yet fixed that will update this, but I don't want to push it yet because I'm waiting on brew to build downstream, and I don't want to risk changing the upstream refs before it finishes.

It will instead look like this:
```
pip = "23.3.1"  # see https://github.com/quipucords/quipucords/pull/2559
```